### PR TITLE
[new] add simplePagination.js@0.0.3

### PR DESCRIPTION
**Pull request by Amomo**
Hi @maruilian11 this PR is for #6527

I add `simplePagination.js` at v0.0.3 **without** auto-update config.
This is a manually addition copying the files from https://github.com/flaviusmatis/simplePagination.js/commit/71468116faf8106d653649ed2582ce6d49bb8013.

Would you mind checking this PR for me? Thank you~ :-D
# Profile of the lib
- Git repository (required): https://github.com/flaviusmatis/simplePagination.js
- Official website (optional, not the repository): http://flaviusmatis.github.io/simplePagination.js/
- GitHub / Bitbucket popularity (required):
  - Count of stars: **534**
  - Count of watchers: 54
  - Count of forks: 250
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [ ] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [ ] Has valid tags for each versions (for git auto-update)
- [ ] Auto-update setup
- [ ] Auto-update target/source is valid.
- [ ] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by maruilian11** [comment]
@Amomo
already exist / duplicated issue and PR?
map file, please.

@PeterDaveHello do we need to add auto-update for future?

**Comment by Amomo** [comment]
@maruilian11 oh #8134 is closed! I don't notice that, sorry.
And, I have attached the map file, could you help me checking again?
Thanks!

**Comment by maruilian11** [comment]
i think @PeterDaveHello may wish we can add auto-update for this lib for future, let's wait for his response.

**Comment by PeterDaveHello** [comment]
Have no idea why you want to assign me for that reason, don't make meaningless works, this will just make my loading higher but help nothing!

**Comment by maruilian11** [comment]
ok, @Amomo 
could you add auto-update config for future?

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

File: ajax/libs/simplePagination.js/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "simplePagination.js",
+  "filename": "jquery.simplePagination.min.js",
+  "description": "A simple jQuery pagination plugin with 3 CSS themes.",
+  "version": "0.0.3",
+  "homepage": "http://flaviusmatis.github.io/simplePagination.js/",
```
Comment (pvnr0082t) [inline ajax/libs/simplePagination.js/package.json]: Let's use `https` for **github.io** domain. Thanks.

Comment (Amomo) [inline ajax/libs/simplePagination.js/package.json]: 🆗

**Comment by sufuf3** [comment]
@Amomo Would you like to finish this PR?
Thank you.

**Comment by PeterDaveHello** [comment]
Close due to no response.

**Pull request closed without merging**